model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/updater/liquibase/MapJpaLiquibaseUpdaterProvider.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.models.map.storage.jpa.updater.liquibase;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.lang.reflect.Method;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import liquibase.Contexts;
+import liquibase.LabelExpression;
+import liquibase.Liquibase;
+import liquibase.changelog.ChangeSet;
+import liquibase.changelog.RanChangeSet;
+import liquibase.exception.LiquibaseException;
+import org.jboss.logging.Logger;
+import org.keycloak.common.util.reflections.Reflections;
+import org.keycloak.connections.jpa.updater.liquibase.conn.LiquibaseConnectionProvider;
+import org.keycloak.connections.jpa.updater.liquibase.ThreadLocalSessionContext;
+import org.keycloak.models.ClientModel;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.map.storage.jpa.updater.MapJpaUpdaterProvider;
+import org.keycloak.models.map.storage.jpa.updater.MapJpaUpdaterProvider.Status;
+
+public class MapJpaLiquibaseUpdaterProvider implements MapJpaUpdaterProvider {
+
+    private static final Logger logger = Logger.getLogger(MapJpaLiquibaseUpdaterProvider.class);
+
+    private static final Map<Class<?>, String> MODEL_TO_CHANGELOG_FILE = new HashMap<>();
+    static {
+        MODEL_TO_CHANGELOG_FILE.put(ClientModel.class, "META-INF/jpa-clients-changelog.xml");
+    }
```

Comment:
Can you leverage newly added https://github.com/keycloak/keycloak/blob/95614e8b4022306ffac328b85130403cc157fb18/model/map/src/main/java/org/keycloak/models/map/storage/ModelEntityUtil.java#L125 rather than introducing this map?